src/python/pants/core/util_rules/source_files.py

```diff
@@ -50,7 +50,7 @@ def __init__(
         self.enable_codegen = enable_codegen
 
 
-@rule
+@rule(desc="Get all relevant source files")
```

Comment:
This verb might be off.

Note that this is a helper rule that can work on many targets at once, it's not the `HydrateSources` rule.